changelogs/fragments/7653-fix-cloudflare-lookup.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - cloudflare_dns - fix cloudflare lookup of SHFP records in ``cloudflare_dns`` module (https://github.com/ansible-collections/community.general/issues/7652).  
```

Just a small improvement - it's already clear from the `cloudflare_dns -` at the beginning that this is about the cloudflare_dns module, so no need to repeat it. Also Cloudflare itself is spelled with a capital C.
```suggestion
  - cloudflare_dns - fix Cloudflare lookup of SHFP records (https://github.com/ansible-collections/community.general/issues/7652).  
```